components/widgets/menu/menu_items/menu_start_trial.tsx

```diff
@@ -33,8 +34,12 @@ const MenuStartTrial = (props: Props): JSX.Element | null => {
     };
 
     const prevTrialLicense = useSelector((state: GlobalState) => state.entities.admin.prevTrialLicense);
-    const isLicensed = prevTrialLicense?.IsLicensed;
-    const show = isLicensed !== undefined && isLicensed !== 'true';
+    const currentLicense = useSelector(getLicense);
+    const isPrevLicensed = prevTrialLicense?.IsLicensed;
+    const isCurrentLicensed = currentLicense?.IsLicensed;
+
+    // Show this CTA if the instance is currently not licensed and has never had a trial license loaded before
+    const show = (isCurrentLicensed !== undefined && isCurrentLicensed !== 'true') && (isPrevLicensed !== undefined && isPrevLicensed !== 'true');
```

Comment:
Cant this be espresed as  x==='false'?

Reply:
+1, let's simplify this expression

Reply:
Hi @pablovelezvidal it's possible that in the redux state we don't have `isLicensed` available while the license is being fetched. Would `x === 'false'` still work well?